doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -44,6 +44,26 @@ Examples
 ========
 
 Below are examples of a variety two dimensional beam bending problems.
+ 
+.. note::
+
+    There is no specific sign convention that must be followed.
+    Given a right handed coordinate system with XYZ coordinates,
+    the beam's length is assumed to be along the positive X axis.
+    The loading forces are directed either in the positive or
+    negative Y direction and the loading moments are about
+    the positive or negative Z direction.
+    The only thing that should be considered while choosing
+    sign onvention is that forces acting along positive
+    Y direction(upward) and moments acting along
+    negative z direction(clockwise) must have the same signs.
+    A user is free to choose its own sign convention as long as these
+    conditions are met. The results will automatically follow
+    the chosen sign convention.
+    In most of the examples used here, forces acting along
+    positve Y direction and moments along negative z direction
+    are assigned positive signs.
```

Comment:
What examples don't make this assumption?

Reply:
In [example 4](https://docs.sympy.org/latest/modules/physics/continuum_mechanics/beam_problems.html#example-4), we have reaction forces negative.
In [example 6](https://docs.sympy.org/latest/modules/physics/continuum_mechanics/beam_problems.html#example-6), we have forces along -Y direction as positive and moment along +z direction as positive.

Reply:
I think the note that I added is sufficient in explaining it. Is there anything else you would like to suggest? 
I think we can also clear out all confusions regarding sign conventions by adding example 1 again but with opposite sign conventions. What do you suggest?

Reply:
ping @moorepants

Reply:
ping @moorepants